apps/admin_api/test/admin_api/v1/controllers/wallet_controller_test.exs

```diff
@@ -69,8 +72,8 @@ defmodule AdminAPI.V1.WalletControllerTest do
       assert is_boolean(pagination["is_first_page"])
     end
 
-    test "returns a list of wallets according to search_term, sort_by and sort_direction" do
-      {:ok, account} = :account |> params_for() |> Account.insert()
+    test "returns a list of wallets according to sort_by and sort_direction" do
+      account = insert(:account)
```

Comment:
Note to self that `insert(:account)` here so the number of wallets are more manageable during these tests.